build-system/tasks/extension-helpers.js

```diff
@@ -206,6 +209,8 @@ function getExtensionsToBuild() {
     extensionsToBuild = dedupe(extensionsToBuild.concat(extensionsFrom));
   }
 
+  extensionsToBuild = dedupe(extensionsToBuild.concat(DEFAULT_EXTENSION_SET));
```

Comment:
This function will read better if you initialize `extensionsToBuild` with `DEFAULT_EXTENSION_SET` on line 196 and then change line 204 to `extensionsToBuild = dedupe(extensionsToBuild.concat(argv.extensions.split(',')))`.